config/config_helpers.go

```diff
@@ -342,36 +355,11 @@ func parseOptionalQuotedParam(parameters string) (string, string, int, error) {
 // foo("a") -> return []string{"foo"}, nil
 // foo("a", "b", "c", "d") -> return []string{"a", "b", "c", "d"}, nil
 func parseParamList(parameters string) []string {
-	trimmedParameters := strings.TrimSpace(parameters)
-	if trimmedParameters == "" {
-		return []string{}
-	}
-
-	matches := oneQuotedParamRegex.FindStringSubmatch(trimmedParameters)
-	if len(matches) > 0 {
-		return matches[1:]
-	}
-
-	matches = listQuotedParamRegex.FindStringSubmatch(trimmedParameters)
-	if len(matches) != 3 {
+	parsed, err := splitParamList(parameters)
+	if err != nil {
 		return []string{}
 	}
-	var trimmedArgs []string
-	trimmedArgs = append(trimmedArgs, matches[1])
-
-	args := matches[2]
-	args = strings.Replace(args, `"`, "", -1)
-
-	parsedArgs := strings.Split(args, ",")
-
-	for _, arg := range parsedArgs {
-		trimmedArg := strings.TrimSpace(arg)
-		if trimmedArg != "" {
-			trimmedArgs = append(trimmedArgs, trimmedArg)
-		}
-	}
-
-	return trimmedArgs
+	return parsed
```

Comment:
Note, with `splitParamList` factored out, this function mainly just handles the error condition.